docs/api-testing/load-testing.md

```diff
@@ -75,14 +71,22 @@ Variables and snippets in the Vault (Company/Project) are used by default, witho
 
 You can set up multiple profiles for each test, from less aggressive to very aggressive, or with varying durations. The only limitations are those inherited from your infrastructure.
 
-Once you have created and saved a load test, the configuration will be saved for future use. To access the list of previously saved configurations for a specific test, click the load test icon in the test list.
+<img src={useBaseUrl('/img/api-testing/create-load-test.png')} alt="The Create Load test page"/>
+
+5. Click **Save and Run**. 
+
+:::note
+Load testing only works with published tests. If you change a published test, and then run a load test without re-publishing it, the load test will be run on the published version, not the updated one.
```

Comment:
@aliciakrebs I think the note should be after point 2. Users can reach the load test page also from within the composer this means they can change the test and select the load test link without publishing it, so it is better they are aware of it at the start, not when they've already done all the things

Reply:
I moved the note to Step 2.